docs/troubleshooting.md

```diff
@@ -24,6 +37,79 @@ The Open vSwitch daemon logs for each `antrea-agent` Pod are also stored on the
 persistent storage of the corresponding node (i.e. the node on which the Pod is
 scheduled), under `/var/log/antrea/openvswitch`.
 
+## Accessing the antrea-controller API
+
+antrea-controller runs as a Deployment, exposes its API via a Service and
+registers an APIService to aggregate into the Kubernetes API. To access the
+antrea-controller API, you need to know the address of it and have credentials
+to access it. There are multiple ways you can do it.
+
+### Using antctl
+
+Typically, `antctl` handles locating and authenticating when it runs in an
+environment with kubeconfig set up. Same as `kubectl`, `antctl` looks for a
+file named config in the $HOME/.kube directory. You can specify other
+kubeconfig files by setting the `--kubeconfig` flag.
+
+For example, you can view internal NetworkPolicy with this command:
+```
+antctl get network-policy
+```
+
+### Using kubectl proxy
+
+As the antrea-controller API is aggregated into the Kubernetes API, you can
+access the specific paths of the latter one. The following command runs
+`kubectl` in a mode where it acts as a reverse proxy for the Kubernetes API and
+handles authenticating.
+```
+# Start the proxy in the background
+kubectl proxy &
+# Access the antrea-controller API path
+curl 127.0.0.1:8001/apis/networking.antrea.tanzu.vmware.com
+```
+
+### Directly accessing the antrea-controller API
+
+If you want to directly access the antrea-controller API, you need to get the
+address of it and pass an authentication token when accessing it, like this:
+```
+# Get the antrea service address
+ANTREA_SVC=$(kubectl get service antrea -n kube-system -o jsonpath='{.spec.clusterIP}')
+# Get the token value of antctl account, you can use any service accounts that have permissions to antrea API.
+TOKEN=$(kubectl get secrets -n kube-system -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='antctl')].data.token}"|base64 --decode)
+# Access antrea API with TOKEN
+curl --insecure --header "Authorization: Bearer $TOKEN" https://$ANTREA_SVC/apis
+```
+
+## Accessing the antrea-agent API
+
+antrea-agent runs as DaemonSet Pod on each Node, exposes its API via a local
+endpoint. There are two ways you can access it.
+
+### Using antctl
+
+To use `antctl` to access the antrea-agent API, you need to get into the
```

Comment:
s/you need to get into the/you need to exec into the

Reply:
done